Index: src/main/java/com/Group3/Patient.java
===================================================================
diff --git a/src/main/java/com/Group3/Patient.java b/src/main/java/com/Group3/Patient.java
--- a/src/main/java/com/Group3/Patient.java	
+++ b/src/main/java/com/Group3/Patient.java	
@@ -16,6 +16,8 @@
     private String email;
     private String password;
 
+    private int age;
+
     public Patient(){
 
     }
